app/e2e/tests/test-setup.ts

```diff
@@ -0,0 +1,83 @@
+import { chromium } from '@playwright/test';
+import type { Page } from '@playwright/test';
+
+async function addEvent(page: Page, eventDate: Date) {
+    const today = new Date();
+    const eventText = today.toLocaleDateString() === eventDate.toLocaleDateString() ? 'Active Test' : 'Upcoming Test';
+
+    // Set Event Details
+    await page.locator('text=New Event').click();
+    await page.locator('text=Title *Title * >> input[type="text"]').fill(`${eventText} Event`);
+    await page.locator('text=Topic *Topic * >> input[type="text"]').fill(`${eventText} Topic`);
+    await page.locator('text=Description *Description * >> input[type="text"]').fill(`${eventText} Description`);
+
+    // Today's Date is stored in variables to make the Start and End Date/Time selectors more readable
+    const todayMonth = today.toLocaleDateString('en-US', { month: 'short' });
+    const todayDate = today.toLocaleDateString('en-US', { day: '2-digit' });
+    const todayYear = today.getFullYear();
+
+    // Set Start Date/Time
+    await page
+        .locator(
+            `text=Start Date & Time *Start Date & Time * >> [aria-label="Choose date\\, selected date is ${todayMonth} ${todayDate}\\, ${todayYear}"]`
+        )
+        .click();
+    await page.locator('[aria-label="calendar view is open\\, go to text input view"]').click();
+    await page
+        .locator('[placeholder="mm\\/dd\\/yyyy hh\\:mm \\(a\\|p\\)m"]')
+        .fill(`${eventDate.toLocaleDateString()} 12:00 am`);
+    await page.locator('text=OK').click();
+
+    // Set End Date/Time
+    await page
+        .locator(
+            `text=End Date & Time *End Date & Time * >> [aria-label="Choose date\\, selected date is ${todayMonth} ${todayDate}\\, ${todayYear}"]`
+        )
+        .click();
+    await page.locator('[aria-label="calendar view is open\\, go to text input view"]').click();
+    await page
+        .locator('[placeholder="mm\\/dd\\/yyyy hh\\:mm \\(a\\|p\\)m"]')
+        .fill(`${eventDate.toLocaleDateString()} 11:59 pm`);
+    await page.locator('text=OK').click();
+    await page.locator('button:has-text("Create")').click();
+}
+
+async function globalSetup() {
+    const browser = await chromium.launch({
+        headless: false,
```

Comment:
Make sure to update headless to true when you are finished debugging since headless is required when running tests in CI pipeline. Same for the teardown.

Reply:
This should probably be a env variable?

Reply:
possibly, I think by default it is headless so if you just remove the arg it would be fine as well.